atlite/convert.py

```diff
@@ -530,21 +532,24 @@ def wind(cutout, turbine, smooth=False, **params):
 
 
 # solar PV
-def convert_pv(ds, panel, orientation, trigon_model="simple", clearsky_model="simple"):
+def convert_pv(
+    ds, panel, orientation, tracking, trigon_model="simple", clearsky_model="simple"
```

Comment:
Can we make `tracking` a keyword argument here?

Reply:
Default to no tracking would be a sensible choice.

Reply:
Just updated it. The no tracking option is the default option (e.g. see third assertion of pv_tracking_test)